ds3-sdk/src/main/java/com/spectralogic/ds3client/metadata/PosixMetadataRestore.java

```diff
@@ -27,20 +26,25 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import static com.spectralogic.ds3client.utils.MetadataKeyConstants.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public class PosixMetadataRestore extends AbstractMetadataRestore {
+import static com.spectralogic.ds3client.metadata.MetadataKeyConstants.KEY_GID;
+import static com.spectralogic.ds3client.metadata.MetadataKeyConstants.KEY_PERMISSION;
+import static com.spectralogic.ds3client.metadata.MetadataKeyConstants.KEY_UID;
 
-    public PosixMetadataRestore(final Metadata metadata, final String filePath, final String localOS, final MetadataRestoreListener metadataRestoreListener) {
+class PosixMetadataRestore extends AbstractMetadataRestore {
+    private static final Logger LOG =LoggerFactory.getLogger(PosixMetadataRestore.class);
```

Comment:
Shouldn't this get the logger for AbstractMetadataRestore not PosixMetadataRestore?